examples/connectivity/compare_resting_state_decomposition.py

```diff
@@ -0,0 +1,81 @@
+"""
+Comparison of Dictionary Learning and ICA for doing group analysis of
```

Comment:
I would write "Group comparisons of Dictionary Learning and Canonical ICA analysis on resting-state fMRI"